### PR TITLE
CaseView: respect visibility condition

### PR DESCRIPTION
**Pull request by JEAxero**
Added logic for the tab component to respect the visibility conditions as configured in Pega.

Note that this logic patch should also be applied to SDK-A and SDK-WC when the change is made to SDK-R

### CODE REVIEW

**Comment by JEAxero** [comment]
There are some lint errors that need to be fixed before we try to merge this change. It may be easier to make the change in a new branch/commit/PR than to edit this git "patch". If we do that, we can decline the PR.

**Comment by rdltev** [comment]
Hi @JEAxero , these linting errors even occur with a fresh install of the ReactSDK. Once those are fixed I will resubmit the request

File: src/components/templates/CaseView/index.tsx [hunk 1]
```diff
@@ -105,8 +105,10 @@ export default function CaseView(props) {
   // populate vertTabInfo and deferLoadInfo
   theTabsRegionChildren.forEach((tabComp, index) => {
     const theTabCompConfig = tabComp.getPConnect().getConfigProps();
-    vertTabInfo.push({name: theTabCompConfig.label, id: index});
-    deferLoadInfo.push( { type: "DeferLoad", config: theTabCompConfig } );
+    if (theTabCompConfig.visibility==true){
```
Comment (JEAxero) [inline src/components/templates/CaseView/index.tsx:108]: The lint errors can be fixed by changing this line to simply:

if (theTabCompConfig.visibility) {

**Comment by JEAxero** [comment]
We can decline this PR since it was superceded by PR 146

**Comment by JEAxero** [comment]
Closing. This fix was handled in another PR.

**Pull request closed without merging**